### PR TITLE
add attachments support in EmailOperator

### PR DESCRIPTION
**Pull request by CooperLuan**
testcase passed in dag

``` py
from airflow.operators import EmailOperator
from airflow.models import DAG
from datetime import datetime, timedelta

seven_days_ago = datetime.combine(datetime.today() - timedelta(7),
                                  datetime.min.time())
args = {
    'owner': 'airflow',
    'start_date': seven_days_ago,
}

dag = DAG(dag_id='email_operator', default_args=args)

job = EmailOperator(
    to='ohmy@cry.cc',
    subject='Daily Report from Cooper(Data Team)',
    html_content='HelloWorld',
    files=['requirements.txt'],
    task_id='email_me',
    dag=dag,
)
```

### CODE REVIEW

File: airflow/operators/email_operator.py [hunk 1]
```diff
@@ -26,11 +28,13 @@ def __init__(
             to,
             subject,
             html_content,
+            files=[],
```
Comment (mistercrunch) [inline airflow/operators/email_operator.py]: That's a python gotcha! never instantiate in default of a method header or all functions calls will refer to the same object id. Instead, go `file=None`, and in the body of the function
`self.files = files or []`

File: airflow/utils.py [hunk 2]
```diff
@@ -353,7 +354,7 @@ def ask_yesno(question):
             print("Please respond by yes or no.")
 
 
-def send_email(to, subject, html_content):
+def send_email(to, subject, html_content, files=[]):
```
Comment (mistercrunch) [inline airflow/utils.py]: same here!

**Comment by mistercrunch** [comment]
This looks great apart from the gotchas I pointed out, more info about it here:
https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/

**Comment by CooperLuan** [comment]
thanks, it should looks ok now

**Comment by mistercrunch** [comment]
img

**Pull request merged**